app/routes.py

```diff
@@ -54,13 +56,21 @@ def darkTheme():
 
 @core.route('/experiments')
 def experiments():
+    
+    user_key = str(uuid.uuid1())
+    user = {'key': user_key}
```

Comment:
why aren't you using `get_ld_user(random=True)` instead of this?

Reply:
Not in particular, i can update this.